server/chat-commands/admin.ts

```diff
@@ -18,6 +18,14 @@ interface ProcessData {
 	cmd: string;
 	cpu?: string;
 	time?: string;
+	ram?: string;
+}
+
+function checkCanUse(context: CommandContext) {
+	const devRoom = Rooms.get('development');
+	if (!(devRoom && Users.Auth.atLeast(devRoom.auth.getDirect(context.user.id), '%'))) {
+		context.checkCan('lockdown');
+	}
 }
```

Comment:
This is a really awkward way to do it, moves the original `checkCan` outside of where we can see it, requires passing around contexts which we really shouldn't be doing, and is also named badly in a way that doesn't mention development at all.

Reply:
```suggestion
function hasDevAuth(user: User) {
	const devRoom = Rooms.get('development');
	return devRoom && Users.Auth.atLeast(devRoom.auth.getDirect(user.id), '%');
}
```